modules/ROOT/pages/design-change-root-file.adoc

```diff
@@ -0,0 +1,75 @@
+= Root Files in Projects in the Text Editor in API Designer
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+In API-specification projects that you create in the text editor and in API-fragment projects, one file is always designated as the "root file". In the *Files* pane, the root file is indicated with a rectangle that is labeled *Root file*. When you publish your project to Anypoint Exchange, the root file sets the context for the parser that validates the files in the project.
+
+For example, suppose that a RAML API-specification project contains two specifications, each including (with the `!include` keyword) a different set of API fragments that are also in the project. When you publish the project to Exchange, all of the files in the project are published. However, Exchange needs to validate the specification and the files that it includes. Without a root file, Exchange cannot ascertain which file to parse first and which set of files to parse.
+
+To take another example, suppose that you want to publish an API-fragment project that contains several fragments, one of which is independent and the remaining files of which are a library fragment and its dependencies. If no root file is specified in the project, Exchange cannot parse the files correctly. In this case, you could either set the independent fragment as the root file, indicating to Exchange that the file can be parsed alone. Or you could set the library file as the root file, indicating to Exchange that parsing should begin with that file and proceed to the dependent fragments.
+
+== Root Files in API-Specification Projects
+
+Every API-specification project includes a default root file. You can switch the root file when there is more than one specification in a project.
+
+=== The Default Root File
+
+When you create an API-specification project, the file that is created by default is set to be the root file:
+
+image:apid-default-root-file-api-spec-project.png[The default root file in an API-specification project named "Orders API".]
+
+<1> The name of the RAML API-specification project is `Orders API`.
+<2> The default file is named `orders-api.raml` and is set as the root file because every API-specification project must have a root file.
+
+== Setting the Root File in an API-Specification Project That Contains More Than One Specification
+
+When you import a specification into a project, API Designer asks whether you want to set the specification as the root file of the project.
```

Comment:
@iamerico , I include screenshots where the UI might not be easy to interpret, where I think a user might need some help. This message is straightforward. However, the message has two problems:
1. "We found file" should be "We found a file".
2. The message should display the name of the file, so that the user knows which file APID is referring to. Without knowing the name, how can the user feel confident that APID has found the right file?

Reply:
All right, we' re going to make those changes before the Release.